app/controllers/departments_controller.rb

```diff
@@ -0,0 +1,57 @@
+class DepartmentsController < ApplicationController
+  before_action :set_department, only: [:show, :edit, :update, :destroy]
+  def index
+    @departments = Department.all
+  end
+
+  def show
+  end
+
+  def new
+    @department = Department.new
+  end
+
+  def edit
+  end
+
+  def create
+    @department = Department.new(department_params)
```

Comment:
current_employee.company.departments.new(department_params)

and in the edit as well